server/models/comments.js

```
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+const schema = mongoose.Schema;
+schema = new mongoose.Schema({
```

Biome (1.9.4)

[error] Can't assign schema because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)